examples/edge_bundling.ipynb

```diff
@@ -0,0 +1,524 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "from dask.distributed import Client\n",
+    "from pandas import DataFrame\n",
+    "\n",
+    "import fastparquet as fp\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "\n",
+    "from datashader.bundling import bundle, nop_bundle\n",
+    "from datashader.colors import inferno, viridis\n",
+    "from datashader.utils import export_image\n",
+    "\n",
+    "import datashader as ds\n",
+    "import datashader.transfer_functions as tf"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "def render_points(df, width=4000, height=4000, cmap=None, bgcolor=None):\n",
+    "    cvs = ds.Canvas(plot_width=width, plot_height=height, x_range=(0, 1), y_range=(0, 1))\n",
+    "    agg = cvs.points(df, 'x', 'y',  ds.count())\n",
+    "    img = tf.shade(agg, cmap=cmap) if cmap else tf.shade(agg)\n",
```

Comment:
I'd just put a default value in the definition of the 'cmap' argument and avoid the extra logic here.